pkgs/tools/admin/proxmove/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "proxmove";
+  version = "1.2";
+
+  src = fetchFromGitHub {
+    owner = "ossobv";
+    repo = "proxmove";
+    rev = "v${version}";
+    sha256 = "sha256-8xzsmQsogoMrdpf8+mVZRWPGQt9BO0dBT0aKt7ygUe4=";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    proxmoxer
+  ];
+
+  doInstallCheck = true;
```

Comment:
`doCheck` in `buildPython*` maps to `doInstallCheck` in a standard derivation, and defaults to `true` already.